testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jackson2/resource/PreferJacksonOverJsonBClientResource.java

```diff
@@ -0,0 +1,43 @@
+package org.jboss.resteasy.test.providers.jackson2.resource;
+
+
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.spi.HttpResponseCodes;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Date;
+
+@Path("/")
+public class PreferJacksonOverJsonBClientResource {
+
+   @GET
+   @Path("core")
+   @Produces(MediaType.APPLICATION_JSON)
+   public Date core() {
+      return new Date();
+   }
+
+   @GET
+   @Path("call")
+   @Produces("text/plain")
+   public String call(@HeaderParam("clientURL") String clientURL) throws Exception {
+
+      ResteasyClient client = (ResteasyClient) ResteasyClientBuilder.newBuilder().build();
```

Comment:
Client client = ClientBuilder.newClient(); is likely enough instead of the line above. And in any case, you need to close the client, for example using a try-finally block